orttraining/orttraining/models/runner/training_runner.h

```diff
@@ -166,6 +166,12 @@ class TrainingRunner {
     // pipeline partition information to do online-partition. If the graph is
     // pre-partitioned, no need to fill this value.
     std::vector<TrainingSession::TrainingConfiguration::CutInfo> pipeline_partition_cut_list;
+    // Alternative for partition. We map each operator's string identifier to
+    // a stage identifier. We identify operators using the name of any of
+    // their outputs. All operators in the graph must be in the domain of this
+    // map.
+    std::map<std::string, int> op_id_to_stage;
```

Comment:
For example, op_id_to_stage["MatMul0"] being 5 means the operator node called "MatMul0" locates on the 6th stage. Note that stage ID is 0-based index.

Reply:
Thanks